package.json

```diff
@@ -57,6 +57,8 @@
     "test:ftr": "scripts/use_node scripts/functional_tests",
     "test:ftr:server": "scripts/use_node scripts/functional_tests_server",
     "test:ftr:runner": "scripts/use_node scripts/functional_test_runner",
+    "test:cypress": "env TZ=America/Los_Angeles NO_COLOR=1 yarn cypress run",
+    "test:cypress:open": "env TZ=America/Los_Angeles NO_COLOR=1 yarn cypress open",
```

Comment:
we recently changed the timezone to be run on UTC: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/9358/files#diff-1a2b52d55ed1bf51fa9d61ccc1c27d91bc7d0af28b49ec0905966edf07e57c18R20